pkg/application/application_suite_test.go

```diff
@@ -1,12 +1,15 @@
 package application
 
 import (
+	"testing"
+
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
-	"testing"
+	"github.com/rs/zerolog"
```

Comment:
Minor:
I would propose to use https://github.com/phuslu/log, since we use it in `developer-platform`.
The interface looks the same, but the config is a bit more versatile.
But I don't insist